### DISCUSSION TITLE
Problemi installazione Hassos

### DISCUSSION THREAD

**Discussion by nicoladallora12** [indomus/forum#2976] [HUB: Home Assistant]
Buongiorno a tutti, sono nuovo e spero di aver fatto giusto scrivendo qui. 
sto installando Hassos su un nuovo mini pc Intel. Ho seguito alla lettera la procedura e sono arrivato al riavvio togliendo la chiavetta usb però il mini pc mi ferma perché dice che è attivo secure boot. Nel bios però non riesco a disabilitarlo in nessun modo. Come posso fare ?
Grazie mille a chi vorrà aiutarmi.

**Comment by snakuzzo** [comment]
Devi smanettare nelle configurazioni del BIOS. Di sicuro ci sarà il modo di farlo.
